{
    "Computer":  "FRWSTH",
    "Events":  [
                   {
                       "TimeCreated":  "\/Date(1759485887193)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start \"\\Microsoft\\Windows\\UpdateOrchestrator\\USO_UxBroker\" task for user \"NT AUTHORITY\\SYSTEM\". Additional Data: Error Value: 2147942402.",
                       "Id":  101,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  185687
                   },
                   {
                       "TimeCreated":  "\/Date(1759485887193)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to launch action \"%systemroot%\\system32\\MusNotification.exe\" in instance \"{d5d4eb2f-ac02-42f0-8584-ec831eb32ac3}\" of task \"\\Microsoft\\Windows\\UpdateOrchestrator\\USO_UxBroker\". Additional Data: Error Value: 2147942402.",
                       "Id":  203,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  185686
                   },
                   {
                       "TimeCreated":  "\/Date(1759483459984)\/",
                       "Source":  "Microsoft-Windows-WindowsUpdateClient",
                       "LogName":  "Microsoft-Windows-WindowsUpdateClient/Operational",
                       "Level":  "Error",
                       "Message":  "Windows Update failed to check for updates with error 0x80240438.",
                       "Id":  25,
                       "ErrorCode":  "0x80240438",
                       "UpdateTitle":  null,
                       "EventRecordID":  17958
                   },
                   {
                       "TimeCreated":  "\/Date(1759467574748)\/",
                       "Source":  "NETLOGON",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "This computer was not able to set up a secure session with a domain controller in domain KHOETECH due to the following: \r",
                       "Id":  5719,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  8770
                   },
                   {
                       "TimeCreated":  "\/Date(1759467574132)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to load task \"\\Microsoft\\Windows\\UpdateOrchestrator\\StartOobeAppsScanAfterUpdate\" at service startup. Additional Data: Error Value: 2147942402.",
                       "Id":  146,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  184645
                   },
                   {
                       "TimeCreated":  "\/Date(1759467574132)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task instantiation failed \"\\Microsoft\\Windows\\UpdateOrchestrator\\StartOobeAppsScanAfterUpdate\". Check point: Ubpm-RegisterConsumer. Error Value: 2147942402.",
                       "Id":  151,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  184644
                   },
                   {
                       "TimeCreated":  "\/Date(1759467565835)\/",
                       "Source":  "iaStorAC",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "Error log: Smart event occured on disk :49QA14SYK00N  ",
                       "Id":  4102,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  8682
                   },
                   {
                       "TimeCreated":  "\/Date(1759467307299)\/",
                       "Source":  "Microsoft-Windows-DistributedCOM",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "DCOM got error \"1115\" attempting to start the service wuauserv with arguments \"Unavailable\" in order to run the server:\r",
                       "Id":  10005,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  8646
                   },
                   {
                       "TimeCreated":  "\/Date(1759467304766)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start instance \"{c25f52ac-e043-4aff-9ac3-be4eec3680a1}\" of \"\\Microsoft\\Windows\\Wininet\\CacheTask\"  task for user \"\" . Additional Data: Error Value: 2147942404.",
                       "Id":  103,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  184610
                   },
                   {
                       "TimeCreated":  "\/Date(1759467304766)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to complete task \"\\Microsoft\\Windows\\Wininet\\CacheTask\" , instance \"{c25f52ac-e043-4aff-9ac3-be4eec3680a1}\" , action \"Wininet Cache task object\" . Additional Data: Error Value: 2147942404.",
                       "Id":  202,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  184609
                   }
               ],
    "LastPoll":  "10/03 06:13"
}